.github/workflows/ci.yml

```diff
@@ -96,43 +106,97 @@ jobs:
         run: cargo +nightly install cbindgen
       - run: ${{ matrix.make_command }}
         shell: bash
-      - if: matrix.channel == 'stable' && contains(github.ref, 'tags/v')
-        uses: actions/upload-artifact@v2
-        with:
-          name: artifact
-          path: dist/wgpu-*.zip
 
-  publish:
-    if: contains(github.ref, 'tags/v')
-    needs: [build]
-    name: Publish release artifacts
-    runs-on: ubuntu-18.04
+  # Builds for binary releases.
+  # Also build on non-tag commits; it also functions as a test.
```

Comment:
I'd slightly prefer to only bother building tagged commits to avoid the potential for CI backlog, but we could start with this and see how it goes